### DISCUSSION TITLE
"AuthenticatesSessions" not found

### DISCUSSION THREAD

**Discussion by fabiopittol** [Laravel-Backpack/community-forum#241] [Q&A (Help)] [answered]
I'm trying to install Backpack on an existing Laravel project (v8.83.23) but I'm getting an error regarding an Illuminate middleware not being found. And indeed, according to the Laravel docs, it doesn't on Laravel 8.

When I try to access /admin I get:
```
Error
Interface "Illuminate\Contracts\Session\Middleware\AuthenticatesSessions" not found 
```

And looking here: https://laravel.com/api/8.x/Illuminate/Contracts/Session.html it really shows that this middleware doesn't exist on v8, only on v9.

Am I doing something wrong during the Backpack install? Do I have to inform it I'm using v8?

**Comment by tabacitu** [comment, marked as answer]
Hey @fabiopittol ,

Indeed we had missed the fact that Laravel 8 does not have that middleware, when we pushed a new security feature.

It's fixed now, so a "composer update" should do it for you.

Cheers!
